src/main/java/io/supertokens/webserver/api/session/SessionRemoveAPI.java

```diff
@@ -66,6 +66,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             }
         }
 
+        Boolean revokeAcrossAllTenants = InputParser.parseBooleanOrThrowError(input, "revokeAcrossAllTenants", true);
+        if (revokeAcrossAllTenants == null) {
+            revokeAcrossAllTenants = true;
+        }
```

Comment:
if this is specified in the input, and the sessionHandles is specified in the input, throw a 400 error

Reply:
Done